test/lib/mayaUsd/render/mayaToHydra/testFlowViewportAPI.py

```diff
@@ -38,6 +38,11 @@ class TestFlowViewportAPI(mtohUtils.MtohTestCase): #Subclassing mtohUtils.MtohTe
     IMAGE_DIFF_FAIL_THRESHOLD = 0.1
     IMAGE_DIFF_FAIL_PERCENT = 2
 
+    @classmethod
```

Comment:
This is not equivalent to what you had before: previously, you did a file new at the end of each test method.  As I suggested in
https://github.com/Autodesk/maya-hydra/pull/75/files/e3184227bf913bbbb1f48fb2e75ad2b8b5944239..eff7ba8689bcfe5db288d514517fba122b1e90a5#r1499561088
the proper equivalent is tearDown():
https://docs.python.org/3/library/unittest.html#unittest.TestCase.tearDown
If you're O.K. with tearDownClass() then that's fine too.

Reply:
You're right, I want to use tearDown.